pakiti-client.spec

```diff
@@ -1,11 +1,12 @@
 Summary:	Patching status monitoring tool
 Name:		pakiti-client
-Version:	X
-Release:	1%{?dist}
+Version:	3.0.3
+%global rel	1
+Release:	%{rel}%{gittag}.ug
```

Comment:
what is the full final rpm version with these?

Reply:
```RPMS/noarch/pakiti-client-3.0.3-11650987211.e50c991.ug.noarch.rpm```

Reply:
thought as much. there is a missing `.` between `%rel ` and `%gittag`. if you have made the rpm and put it in the repos, you need to bump the version to 3.0.4

Reply:
https://github.com/hpcugent/slurm-spank-talamini/blob/master/slurm-spank-talamini.spec#L7 looks ok

Reply:
my bad, it's fixed now.